src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -153,7 +154,9 @@ func applyClientConfig(client *client.Client, cfg *config.Config, deviceID strin
 	} else {
 		fronted.Configure(certs, cfg.Client.MasqueradeSets, filepath.Join(appdir.General("Lantern"), "masquerade_cache"))
 	}
-	logging.Configure(cfg.CloudConfigCA, deviceID, Version, RevisionDate, cfg.BordaReportInterval, cfg.BordaSamplePercentage, cfg.LogglySamplePercentage)
+	configureLoggingOnce.Do(func() {
```

Comment:
Yeah that's cool. I might consider removing all those intervals from the config in that case since they won't do anything but not a biggie